.pre-commit-config.yaml

```diff
@@ -37,7 +37,7 @@ repos:
   - id: isort
 
 - repo: https://github.com/psf/black
-  rev: 21.12b0
+  rev: 22.8.0
```

Comment:
I also had to update this otherwise I was hitting [this](https://github.com/psf/black/issues/2964) issue. After this change, I was able to run `make check` without any issues.